src/Masa.Utils.Exceptions/Handlers/GlobalExceptionFilter.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Utils.Exceptions.Handlers;
+
+/// <summary>
+/// Mvc pipeline exception filter to catch global exception
```

Comment:
If it only use for Mvc, maybe rename MvcGlobalExcetionFilter is better?

Reply:
Processed